internal/http/web/handler.go

```diff
@@ -258,3 +342,52 @@ func staticFileReplace(mode string) func(path string) string {
 		return "/404"
 	}
 }
+
+// rollbarReportError reports an error to rollbar and logs it locally.  It
+// should only be reporting errors in production.  You should not call this
+// function directly, instead call h.Error(...) and let that handle it.
+func rollbarReportError(w http.ResponseWriter, r *http.Request, interfaces ...interface{}) {
+	rollbar.Error(interfaces...)
+	rollbar.Wait()
+
+	// Also log the error locally
+	log.Println("ERROR:", interfaces)
+}
```

Comment:
Do we need to send in the response writer and request to this function if we're not using it?

Reply:
Good catch, removed